Userland/js.cpp

```diff
@@ -62,10 +62,10 @@ class ReplObject : public JS::GlobalObject {
     static JS::Value save_to_file(JS::Interpreter&);
 };
 
-bool dump_ast = false;
-bool print_last_result = false;
-static OwnPtr<Line::Editor> editor;
-static int repl_line_level = 0;
+static bool s_dump_ast = false;
```

Comment:
I think `g_...` would make more sense here.

Reply:
And then change it to non-`static`? A search for `static .* g_` has only 6 results across the codebase...

Reply:
The coding style says static members should be prefixed with s_ but these are global variables and not members. Maybe @awesomekling  should have a look at it.

Reply:
Yeah the `s_` prefix convention is really only for static data members, but I see we have a lot of static locals prefixed with `s_` as well. Accidental habit I guess :) This convention makes sense to me, so let's roll with it.